### PR TITLE
rt98.py: fix check_ver() error - fixes #10833

### PR DESCRIPTION
**Pull request by KC9HI**
# CHIRP PR Checklist

The following must be true before PRs can be merged:

* All tests must be passing.
* Commits should be squashed into logical units.
* Commits should be rebased (or simply rebase-able in the web UI) on current master. Do not put merge commits in a PR.
* Commits in a single PR should be related.
* Major new features or bug fixes should reference a [CHIRP issue](https://chirp.danplanet.com/projects/chirp/issues).
* New drivers should be accompanied by a test image in `tests/images` (except for thin aliases where the driver is sufficiently tested already).
* All files must be GPLv3 licensed or contain no license verbiage. No additional restrictions can be placed on the usage (i.e. such as noncommercial).

Please also follow these guidelines:

* Keep cleanups in separate commits from functional changes.
* Please write a reasonable commit message, especially if making some change that isn't totally obvious (such as adding a new model, adding a feature, etc).
* Do not add new py2-compatibility code (No new uses of `six`, `future`, etc).
* All new drivers should set `NEEDS_COMPAT_SERIAL=False` and use `MemoryMapBytes`.
* New drivers and radio models will affect the Python3 test matrix. You should regenerate this file with `tox -emakesupported` and include it in your commit.

### CODE REVIEW

File: chirp/drivers/retevis_rt98.py [hunk 1]
```diff
@@ -586,7 +586,7 @@ def _ident(radio):
     ver_response = radio.pipe.read(16)
     LOG.debug(util.hexprint(ver_response))
 
-    verok, model, bandlimit = check_ver(radio, ver_response,
+    verok, model, bandlimit = check_ver(ver_response,
```
Comment (kk7ds) [inline chirp/drivers/retevis_rt98.py:589]: Hmm, yeah @dforsi changed this recently in a big cleanup, but it seems clearly wrong. Maybe we can get a read on what the intent was? Definitely looks like that was wrong and this is right though.

**Comment by KC9HI** [comment]
I didn't try to check what happened. I thought that I may have unintentionally changed it some how and didn't follow up the test with upload/download using a physical radio. I went back to the CHIRP legacy driver to see what it was matched what I thought it should and made the "correction". But if @dforsi wants to look into it and submit a fix, that is fine with me.

**Comment by kk7ds** [comment]
No, I think this is the right fix, I just want to know why the change amidst a bunch of otherwise typo-like fixes.

**Comment by KC9HI** [comment]
Oh yeah... now I remember the "big cleanup" that you are talking about. Mostly cosmetic things. This change wasn't cosmetic, though.

**Comment by kk7ds** [comment]
Right, which is why I just want @dforsi to comment.

**Comment by dforsi** [comment]
My commit was about undefined names (commit https://github.com/kk7ds/chirp/commit/90cdd61ba2666c05882683ca863934c702d1fe80) but was wrong because somehow I missed to commit a change of the method signature.
I was trying to fix line 591 at that time, which is now line 571 and still needs a fix, the line is `_finish(radio)` but `radio` is undefined inside `check_ver()`.

**Pull request merged**